content/ohai.md

```diff
@@ -196,21 +191,19 @@ windows
 
 ## Optional Plugins
 
-Ohai ships several plugins that are considered optional and can be
-enabled in the [client.rb configuration file](/config_rb_client/).
+Ohai ships several optional plugins that you can enable in the [client.rb configuration file](/config_rb_client/).
 
 - `:IPC` - SysV IPC shmem information (New in Chef Infra Client 16)
 - `:Interupts` - Data from /proc/interrupts and /proc/irq (New in Chef Infra Client 16)
 - `:Lspci` - PCI device information on Linux hosts.
 - `:Lsscsi` - SCSI device information on Linux hosts.
-- `:Passwd` - User and Group information on non-Windows hosts. This plugin can result in very large node sizes if a system connects to Active Directory or LDAP.
+- `:Passwd` - User and Group information on non-Windows hosts. This plugin can result in large node sizes if a system connects to Active Directory or LDAP.
 - `:Sessions` - Sessions data from loginctl on Linux hosts.
 - `:Sysctl` - All sysctl values on Linux hosts.
```

Comment:
We're missing a one more here:

Grub2: Information from the Grub2 bootloader on Linux